packages/xstate-react/test/useSelector.test.tsx

```diff
@@ -564,4 +571,32 @@ describeEachReactMode('useSelector (%s)', ({ suiteKey, render }) => {
 
     expect(renders).toBe(suiteKey === 'strict' ? 2 : 1);
   });
+
+  it('should compute a stable snapshot internally when selecting from uninitialized service', () => {
+    const child = createMachine({});
+    const machine = createMachine({
+      invoke: {
+        id: 'child',
+        src: child
+      }
+    });
+
+    const snapshots: AnyState[] = [];
+
+    function App() {
+      const service = useInterpret(machine);
+      useSelector(service, (state) => {
+        snapshots.push(state);
+        return state.children.child;
+      });
+      return null;
+    }
+
+    console.error = jest.fn();
+    render(<App />);
+
+    const [snapshot1] = snapshots;
+    expect(snapshots.every((s) => s === snapshot1));
+    expect(console.error).toHaveBeenCalledTimes(0);
```

Comment:
I'd love to find a more robust/flexible way of specifying the logging/error sinks for v5 (future note, not for this PR)

Reply:
Note that this doesn't test a log that we control. This one comes from React